src/datahandlers/complexportal.py

```diff
@@ -0,0 +1,31 @@
+from src.babel_utils import pull_via_urllib, make_local_name
+from src.prefixes import COMPLEXPORTAL
+
+def pull_complexportal():
+    pull_via_urllib('http://ftp.ebi.ac.uk/pub/databases/intact/complex/current/complextab/',f'559292.tsv', decompress=False, subpath=COMPLEXPORTAL)
+
+def make_labels_and_synonyms(infile, labelfile, synfile):
+    infile = open(infile, 'r')
+    lines = infile.readlines()[1:]
+    usedsyns = set()
+
+    with open(labelfile, 'w') as outl, open(synfile, 'w') as outsyn:
+        for line in lines:
+            sline = line.split("\t")
+            id = sline[0]
+            label = sline[1] # recommended name
+            outl.write(f'Complex ac:{id}\t{label}\n')
```

Comment:
This needs to be updated per the biolink prefix.

Reply:
have we requested its addition?

Reply:
Either way, we should replace it here with the constant from prefixes.py